WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt

```diff
@@ -166,26 +169,29 @@ class PagesViewModel
         val post: PostModel,
         val previewType: RemotePreviewType
     )
-
```

Comment:
NitPick: We might want to revert this change.

Reply:
Reverted!